detox/src/devices/Device.js

```diff
@@ -42,6 +43,16 @@ class Device {
     await this.client.sendUserNotification(params);
   }
 
+  async setOrientation(orientation) {
+    // orientation is 'landscape' (meaning left side portrait) or 'portrait' (non-reversed)
+    const call = invoke.call(
```

Comment:
I spent try my code, it's reasonable I mistaken.
What is the working syntax then?